repository/l.json

```diff
@@ -1391,6 +1391,17 @@
 			]
 		},
 		{
+			"name": "LSCheckstyle",
+			"details": "https://github.com/kauboy26/LimeStyle",
+			"readme": "https://github.com/kauboy26/LimeStyle/blob/master/README.md",
```

Comment:
The URL to the readme must be direct. You can just remove this line and PC will pick up the correct URL regardless.